src/snowcli/cli/snowpark/package/manager.py

```diff
@@ -62,7 +62,9 @@ def upload(file: Path, stage: str, overwrite: bool):
 
 def create(zip_name: str):
     file_name = zip_name if zip_name.endswith(".zip") else f"{zip_name}.zip"
-    zip_dir(dest_zip=Path(file_name), source=Path.cwd())
+    zip_dir(
+        dest_zip=Path(file_name), source=Path(os.path.join(os.getcwd(), ".packages"))
```

Comment:
```suggestion
        dest_zip=Path(file_name), source=Path.cwd() / ".packages")
```
This should work too